src/codegen/mod.rs

```diff
@@ -618,6 +618,10 @@ impl CodeGenerator for Var {
             return;
         }
 
+        if let Some(comment) = item.comment(ctx) {
+            result.push(attributes::doc(comment));
+        }
```

Comment:
There's some places here were we could still bail out without generating the constant. Instead, let's move the `let mut attrs = vec![];` from below to here, and append to `attrs`.

With that fixup, this looks great, thanks!